wagtail/admin/tests/test_admin_search.py

```diff
@@ -107,3 +109,78 @@ def test_search_other(self):
 
         self.assertNotIn("Pages", rendered)
         self.assertIn("My Search", rendered)
+
+
+class TestCase(BaseSearchAreaTestCase):
+    """Tests the comparison functions."""
+
+    def setUp(self):
+        self.search_area1 = SearchArea("Label 1", "/url1", order=100)
+        self.search_area2 = SearchArea("Label 2", "/url2", order=200)
+        self.search_area3 = SearchArea("Label 1", "/url3", order=300)
+
+    def test_eq(self):
+        self.assertEqual(
+            self.search_area1, self.search_area1
+        )  # Same object, should be equal
```

Comment:
Can we use `self.assertTrue` and `self.assertFalse` with the `==` operator instead? e.g.

```suggestion
        # Same object, should be equal
        self.assertTrue(self.search_area1 == self.search_area1)
```

The `assertEqual` method of `TestCase` seems to have some magic underneath it. Testing it with `assertTrue`/`assertFalse` would be more explicit for this case, as we know that we are actually testing the operators instead of relying on `assertEqual`'s implementation.